encrypted-media-respec.html

```diff
@@ -1876,8 +1886,18 @@
           <p>The following steps are run:</p>
           <ol>
             <li><p>Let the <var>session</var> be the associated <a>MediaKeySession</a> object.</p></li>
+            <li>
+              <p>
+                Destroy any license and/or keys associated with <var>session</var> that have not been explicitly stored.
+              </p>
+            </li>
             <li><p>Run the <a def-id="update-key-statuses-algorithm"></a> algorithm on the <var>session</var>, providing an empty sequence.</p></li>
             <li><p>Run the <a def-id="update-expiration-algorithm"></a> algorithm on the <var>session</var>, providing <code>NaN</code>.</p></li>
+            <li>
+              <p>
+                If the object's <var>session type</var> is <a def-id="persistent-usage-record-session"></a>, store the <var>record of key usage</var>.
```

Comment:
minor: Is there a reason you added this step after the two above? It seems like the algorithm should do all the internal work then do the externally-visible steps.

I guess this is consistent with the order in remove(). Perhaps we should reorder both in a separate commit.